tests/integration/records/test_views.py

```diff
@@ -186,6 +186,16 @@ def test_literature_facets(api_client, db, create_record):
     assert expected_facet_keys == response_data_facet_keys
 
 
+def test_literature_facets_author_count_does_not_have_empty_bucket(
+    api_client, db, create_record
+):
+    response = api_client.get("/literature/facets/")
+    response_data = json.loads(response.data)
+    response_status_code = response.status_code
+    author_count_agg = response_data.get("aggregations")["author_count"]
+    assert author_count_agg["buckets"] == []
+
```

Comment:
maybe you could also add a test in which the author_count is more than 0 and check that the bucket is there?